src/cls/lock/cls_lock_types.h

```diff
@@ -14,40 +14,40 @@
 #define LOCK_FLAG_MUST_RENEW 0x2   /* lock must already be acquired */
 
 enum ClsLockType {
-  LOCK_NONE                = 0,
-  LOCK_EXCLUSIVE           = 1,
-  LOCK_SHARED              = 2,
-  LOCK_EXCLUSIVE_EPHEMERAL = 3, /* lock object is removed @ unlock */
+  C_LOCK_NONE                = 0,
```

Comment:
Nit: that's pretty ugly. Can we change these to `LOCK_TYPE_xyz` instead?

Reply:
I guess this is to prevent ID clash with names used in WIN32 headers?
Alternative would be to make it `CLS_LOCK_....`